examples/webpack.config.local.js

```diff
@@ -46,7 +46,16 @@ const LOCAL_DEVELOPMENT_CONFIG = {
 };
 
 function addLocalDevSettings(config) {
-  Object.assign(config.resolve.alias, LOCAL_DEVELOPMENT_CONFIG.resolve.alias);
+  if (config.resolve) {
```

Comment:
Idea: If we added this line and we could remove the big if statement:
```
config.resolve = config.resolve || {}
```
We could do the same for rules (although needs to be initialized to epmty array, not object.